csharp/ql/lib/semmle/code/csharp/controlflow/internal/ControlFlowGraphImpl.qll

```diff
@@ -1288,7 +1288,7 @@ module Statements {
     }
 
     final override predicate first(ControlFlowElement first) {
-      // Unlike most other statements, `foreach` statements are not modelled in
+      // Unlike most other statements, `foreach` statements are not modeled in
```

Comment:
Out of curiosity - are we using American spelling over UK spelling?

Reply:
Yes: https://github.com/github/codeql/blob/main/docs/query-help-style-guide.md#english-style

Reply:
Yes. 
https://github.com/github/codeql/blob/main/docs/qldoc-style-guide.md#language-requirements

Reply:
That's about the help, but it would be odd if that wouldn't extend elsewhere.

Reply:
Makes sense 😄

Reply:
Those pages are for qldoc comments and qhelp pages.  Strictly speaking I don't think we require US English in non-QLDoc code comments, though its probably best for consistency.